examples/playground/.stylelintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "stylelint-config-standard",
+  "plugins": [
+    "stylelint-order"
+  ],
+  "rules": {
+    "order/order": [
+      "custom-properties",
+      "declarations"
+    ],
+    "order/properties-alphabetical-order": true
+  }
+}
```

Comment:
Good idea !
Installation and configuration of stylelint should be in a dedicated commit.